--- a/hid-playstation.c	2022-03-13 10:35:10.276668255 +0000
+++ b/hid-playstation.c	2022-03-13 10:36:17.901202246 +0000
@@ -476,12 +476,12 @@
 	for (i = 0; i < ARRAY_SIZE(ps_gamepad_buttons); i++)
 		input_set_capability(gamepad, EV_KEY, ps_gamepad_buttons[i]);
 
-#if IS_ENABLED(CONFIG_PLAYSTATION_FF)
+//#if IS_ENABLED(CONFIG_PLAYSTATION_FF)
 	if (play_effect) {
 		input_set_capability(gamepad, EV_FF, FF_RUMBLE);
 		input_ff_create_memless(gamepad, NULL, play_effect);
 	}
-#endif
+//#endif
 
 	ret = input_register_device(gamepad);
 	if (ret)
